### DISCUSSION TITLE
Project OBC data on Halo

### DISCUSSION THREAD

**Discussion by spambas** [f1multiviewer/issue-tracker#220] [Feature requests] [answered]
In the mainstream broadcast sometimes the OBC data is projected on the Halo. 
Could this be recreated? 

![image](https://user-images.githubusercontent.com/10828880/191476525-5eea8bf9-34d0-43f4-9fb2-e492b4ed6b91.png)

We do have the issue of not being sure the OBC shows the top cam so the Halo will not always be in the OBC.
Also the position of the Halo in the OBC is not always the same. This can be offset by the F1MV army like the sync offset.

**Comment by ManCaveMedia** [comment]
Maybe this can help:
[CSS-Tricks: Curved text along a path](https://css-tricks.com/snippets/svg/curved-text-along-path/)

Others suggested:
- [CSS-Tricks: Set text on a circle](https://css-tricks.com/set-text-on-a-circle/)
- [Warp SVG Online](https://pavellaptev.github.io/warp-svg/)

**Comment by jeppeolesen** [comment, marked as answer]
Closing this as wont fix due to the reasons you mentioned yourself: The onboards aren't always the view that includes the halo - it can also be the rear camera, the driver facing camera or the nose camera - trying to build a solution that accounts for this is not feasible.

If it's something you'd like to experiment with building an integration for, you can explore the MultiViewer internal API, by opening the app and visiting http://localhost:10101/api/graphql in your browser of choice to view the Graph schema, that could allow you to make something like this.
